lightning/src/routing/scoring.rs

```diff
@@ -898,6 +898,9 @@ mod approx {
 	const LOWER_BITS: u32 = 6;
 	const LOWER_BITS_BOUND: u64 = 1 << LOWER_BITS;
 	const LOWER_BITMASK: u64 = (1 << LOWER_BITS) - 1;
+	/// The rough cutoff at which our precision falls off and we should stop bothering to try to
+	/// log a ratio, as X in 1/X.
+	pub(super) const PRECISION_LOWER_BOUND_DENOMINATOR: u64 = LOWER_BITS_BOUND;
```

Comment:
Could you make `LOWER_BITS_BOUND` `pub(super)` and define `PRECISION_LOWER_BOUND_DENOMINATOR` after `NEGATIVE_LOG10_UPPER_BOUND` instead since this is not used in the `approx` module? It will prevent its usage from exceeding the 100 char limit, too.